.github/workflows/release_scripts/whitesource_vulnurability_checker.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import json
+import os
+import boto3
+
+WS_REST_API_URL = 'https://saas.whitesourcesoftware.com/api'
+WS_API_KEY = os.getenv('WS_APIKEY')
+WS_PROJECT_TOKEN = os.getenv('WS_PROJECTTOKEN')
+WS_BLOCKING_VULNERABILITIES = "critical,high"
+
+def determine_any_exclusions():
```

Comment:
suggestion to rename this method to something like get_exclusions() or get_excluded_libraries() since it just gets exclusion entries and returns a list of ,ie, there is no rule or criteria to determine exclusions here